### PR TITLE
Change Font-Family to Poppins/Roboto

### PR DESCRIPTION
**Pull request by jedtan**
This should be on dev for testing!

### CODE REVIEW

File: src/components/groupImpactComponents/GroupGoalNotification.js [hunk 1]
```diff
@@ -10,29 +10,21 @@ const useStyles = makeStyles((theme) => ({
...
   nextGoalButton: {
     background: '#29BEBA',
     borderRadius: '15px',
     height: '30px',
     fontWeight: '900',
-    fontFamily: 'Poppins',
     marginLeft: theme.spacing(1),
     color: 'white',
   },
-  title: {
```
Comment (jedtan) [inline src/components/groupImpactComponents/GroupGoalNotification.js:31]: Not being used anyways.

File: src/utils/theme.js [hunk 2]
```diff
@@ -36,7 +36,19 @@ const defaultThemeObject = {
   },
   typography: {
     fontSize: 14,
-    fontFamily: '"Helvetica Neue", Helvetica, Arial, sans-serif',
+    fontFamily: 'Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif',
```
Comment (kmjennison) [inline src/utils/theme.js:39]: We'll need to also add font source for Roboto as we have for Poppins:
https://mui.com/material-ui/react-typography/#install-with-npm

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: src/utils/theme.js (hunk 2 above)
Comment (jedtan) [reply src/utils/theme.js:39]: Good catch, thanks!

**Comment by jedtan** [comment]
Most recent changes are also on dev.

**Review by kmjennison — APPROVED** [review]
LGTM! ☕

**Pull request merged**